modules/cert-manager-certificate-api-server.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-creating-certificate.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cert-manager-certificate-api-server_{context}"]
+= Creating certificates for API server
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have installed {cert-manager-operator} 1.13.0 or above.
+
+.Procedure
+
+. Create an issuer. For more information, see "Configuring an issuer" in "Additional Resources" section.
+
+. Create a certificate:
+
+.. Create a YAML file, for example, `certificate.yaml`, that defines the `Certificate` object:
++
+.Example `certificate.yaml` file
++
+[source, yaml]
+----
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: <tls_cert> #<1>
+  namespace: openshift-config
+spec:
+  isCA: false
+  commonName: "api.<cluster_base_domain>" #<2>
+  secretName: <tls_cert> #<3>
```

Comment:
If this value should or can be different from the value of <1>, I'd suggest changing the input value name to avoid confusion. This is one possibility:
```suggestion
  secretName: <secret_name> #<3>
```

If this value should or must be the same as <1>, I'd specify that in the callout label on line 43.

Reply:
Great catch, Seb! Let me get this confirmed.

Reply:
Incorporated your suggestion post discussion.